easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-SIRIUS-6.5-rc4-CrayGNU-19.10-cuda-10.1.eb

```diff
@@ -0,0 +1,46 @@
+# created by Anton Kozhevnikov (CSCS)
+# modified by Simon Pintarelli (CSCS)
+easyblock = "ConfigureMake"
+
+name = 'QuantumESPRESSO-SIRIUS'
+version = '6.5-rc4'
+cudaversion = 10.1
+versionsuffix = '-cuda-%s' % cudaversion
```

Comment:
```suggestion
version = '6.5'
local_siriusversion='6.5.2'
local_cudaversion = 10.1
versionsuffix = '%s-cuda-%s' % (local_siriusversion, local_cudaversion)
```

Reply:
You need to fetch the specific version of QE with SIRIUS from this list of releases https://github.com/electronic-structure/q-e-sirius/releases